index1.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+
+<body bgcolor=white>
```

Comment:
It is ill-advised to use markup to affect presentation. Styling should be moved to the CSS to keep concerns properly separated. The general rule to abide by: HTML specifies _what_ data to display, while CSS says _how_ to display it.